### PR TITLE
use only virsh define to bring up libvirt instances

### PR DESCRIPTION
**Pull request by p3ck**
Previously, depending if you defined cloud-config in your topology
the libvirt provisioner would either use virt-install or virt ansible
module. This patch eliminates the virt-install path and consolidates
on just using the ansible virt module.

### CODE REVIEW

File: config/Dockerfiles/run_tests.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ shift
 DISTROS=$*
 #TARGETS="dummy os-server-new os-sg-new os-vol-new"
 
-PROVIDERS="dummy openstack"
+PROVIDERS="dummy libvirt"
```
Comment (14rcole) [inline config/Dockerfiles/run_tests.sh]: Was this supposed to be a temporary change, or was having openstack there wrong to begin with?

Comment (p3ck) [reply config/Dockerfiles/run_tests.sh]: I believe it was a mistake to begin with.  The PROVIDERS in run_tests.sh don't affect what we run in jenkins, only when running locally with make test.  libvirt will work locally, the openstack provider here will only work if you have a openstack clouds.yaml in the keys directory.

**Comment by samvarankashyap** [comment]
@p3ck : Checking whether is changes anything related to virt-customize

**Comment by samvarankashyap** [comment]
@p3ck Needs rebase .

**Comment by p3ck** [comment]
[merge]

**Pull request merged**